LandonHotel/Services/BookingService.cs

```diff
@@ -33,8 +33,13 @@ public bool IsBookingValid(int roomId, Booking booking)
 
         public double CalculateBookingCost(int roomId, Booking booking)
         {
-            //TODO: Booking cost is the adjusted room rate times the number of days
-            return 0.0;
+            double numberdays = 2;//booking.CheckOutDate - booking.CheckInDate;
```

Comment:
Missing the refactor step that removes the constant values; instead use the date difference with and the DateTime property .TotalDays to calculate this